src/PTVS/Microsoft.PythonTools.Analyzer/Impl/AnalyzerScrapers.pyproj

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <SchemaVersion>2.0</SchemaVersion>
+    <ProjectGuid>{1fc96711-4e42-4c41-b7fc-b4141d57d0bb}</ProjectGuid>
+    <ProjectHome />
+    <StartupFile>PythonScraper.py</StartupFile>
+    <SearchPath />
+    <WorkingDirectory>.</WorkingDirectory>
+    <OutputPath>.</OutputPath>
+    <ProjectTypeGuids>{888888a0-9f3d-457c-b088-3a5042f75d52}</ProjectTypeGuids>
+    <LaunchProvider>Standard Python launcher</LaunchProvider>
+    <InterpreterId>{2af0f10d-7135-4994-9156-5d01c9c11b7e}</InterpreterId>
+    <InterpreterVersion>2.7</InterpreterVersion>
+    <CommandLineArguments>"C:\Users\steve_000\AppData\Local\Python Tools\CompletionDB\Debug\12.0\2af0f10d-7135-4994-9156-5d01c9c11b7e\2.7" "C:\USERS\STEVE_000\APPDATA\LOCAL\MICROSOFT\VISUALSTUDIO\12.0EXP\EXTENSIONS\MICROSOFT\PYTHON TOOLS FOR VISUAL STUDIO\2.1\CompletionDB"</CommandLineArguments>
```

Comment:
that looks like something that shouldn't be there

Reply:
It must have been so Steve can more easily use PTVS to edit the various scraper files. Looks like it can be deleted.